src/main/java/com/webapp/TextBook/controller/web_controller/HomeController.java

```diff
@@ -1,42 +1,118 @@
 package com.webapp.TextBook.controller.web_controller;
 
 
+import com.webapp.TextBook.repository.adapter.Adapter;
+import com.webapp.TextBook.repository.data_access.User;
+import com.webapp.TextBook.repository.data_access.UserRole;
+import com.webapp.TextBook.sharedFiles.SharedSessionData;
+import com.webapp.TextBook.sharedFiles.StatusCode;
+import org.javatuples.Pair;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.configurationprocessor.json.JSONArray;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.ModelMap;
 import org.springframework.validation.BindingResult;
-import org.springframework.validation.Errors;
-import org.springframework.validation.FieldError;
 import org.springframework.validation.ObjectError;
 import org.springframework.web.bind.annotation.*;
 
-import java.text.DateFormat;
-import java.text.ParseException;
-import java.text.SimpleDateFormat;
 import java.util.*;
 
 
-import org.springframework.http.MediaType;
-
-import javax.sound.midi.SysexMessage;
 import javax.validation.Valid;
 
-import org.springframework.stereotype.Repository;
-import org.springframework.transaction.annotation.Transactional;
-
-import javax.persistence.EntityManager;
-import javax.persistence.PersistenceContext;
-import javax.persistence.Query;
-import java.util.List;
+import com.webapp.TextBook.viewModel.sharedViewModel.loginUserInfo.LoginUserInfo;
 
 @Controller
-@RequestMapping(path = "/")
+@RequestMapping(path = "/home/")
 public class HomeController {
 
+
+    // fetching shared adapter for encapsulated, database interaction
+    @Autowired Adapter adapter;
+
+    //login page
     @RequestMapping(value = "/", method = RequestMethod.GET)
+    public String login(){ return "login"; }
```

Comment:
the login page is in index.jsp
you may want to change the return "login"  to return "index"